content-src/asrouter/asrouter-content.jsx

```diff
@@ -377,7 +377,7 @@ export class ASRouterUISurface extends React.PureComponent {
             sendUserActionTelemetry={this.sendUserActionTelemetry}
             executeAction={ASRouterUtils.executeAction}
             dispatch={this.props.dispatch}
-            onBlock={this.onBlockById(this.state.message.id)}
+            onBlockById={ASRouterUtils.blockById}
```

Comment:
Hmm.... I do see that ASRouterUtils typically uses `ASRouterUtils` instead of `this` to avoid binding. Although there is a `theme: this.getPreviewTheme()` but this call is okay.